packages/utils/src/browser.ts

```diff
@@ -74,16 +74,22 @@ function _htmlElementAsString(el: unknown): string {
   }
 
   out.push(elem.tagName.toLowerCase());
-  if (elem.id) {
-    out.push(`#${elem.id}`);
-  }
 
-  // eslint-disable-next-line prefer-const
-  className = elem.className;
-  if (className && isString(className)) {
-    classes = className.split(/\s+/);
-    for (i = 0; i < classes.length; i++) {
-      out.push(`.${classes[i]}`);
+  const keyAttrValue = keyAttr ? elem.getAttribute(keyAttr) : null;
```

Comment:
Would anyone be open to the possibility of `serializeAttribute` being an array instead? This would allow for recognizing multiple attributes that are possibly not available on all elements.

For example,

```js
{ 
  serializeAttribute: [
    'data-testid', // all our good devs using react-testing library
    'aria-label', // attempt to fallback on something recognizable
}
...
const keyAttrValue = Array.isArray(keyAttr) && keyAttr.length 
  ? keyAttr.find(attr => elem.getAttribute(keyAttr))
  : null;
```